.circleci/config.yml

```diff
@@ -78,15 +66,14 @@ jobs:
           at: workspace
       - run:
           name: Run unit tests, system tests, jsdoc generation, and gts fix of the generated Showcase library
-          command: |
-            cd workspace/workspace/gapic-generator-typescript/.test-out-showcase
+          working_directory: /home/circleci/project/workspace/gapic-generator-typescript/.test-out-showcase
+          command: &alltests |
             npm install
             npm test
             npm run fix
             npm run compile
             npm run system-test
             npm run docs
-            npm run docs-test
```

Comment:
Yes.  I think it does not make sense to check if links in the generated docs are valid because if they are not, it's more likely not a generator failure.  We disabled this check for DLP and might also want to disable it for monitoring, which probably means it's better to disable it for all libraries.